task/summary/0.2/MIGRATION.md

```diff
@@ -0,0 +1,9 @@
+# Migration from 0.1 to 0.2
+
+The parameter `PIPELINERUN_NAME` used by `summary` task was removed.
```

Comment:
If this parameter can be useful in the future, I'd leave it.

Reply:
ok I'll leave that be